src/js/markdownlint.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+var fluid = require("infusion");
+var path = require("path");
+var markdownlint = require("markdownlint");
+
+fluid.require("fluid-glob");
+
+fluid.lintAll.markdownlint = function (options) {
```

Comment:
Note the incredible similarity between this preamble and the one in fluid.lintAll.mdjsonlint, supporting my idea that these should really be components

Reply:
I didn't quite go as far to factor out a grade for the file globbing, but I did consolidate a lot of things especially in rollup checks.